tests/cypress/e2e/unit_tests/p1_fleet.spec.ts

```diff
@@ -270,6 +270,73 @@ if (!/\/2\.7/.test(Cypress.env('rancher_version'))) {
   });
 }
 
+// RepoURLRegex is supported on v2.8 but error reporting is not working correctly there
+// Ref. https://github.com/rancher/fleet/issues/2462 but it wont be fixed in v2.8
+if (/\/2\.9/.test(Cypress.env('rancher_version'))) {
+  describe('Private Helm Repository tests (helmRepoURLRegex)', { tags: '@p1'}, () => {
+
+    // So far using custom repo, we should expose fleet.yaml somewhere on fleet-qa-examples repo
+    const repoUrl = 'https://github.com/fleetqa/fleet-qa-examples-public.git'
+    const branch = 'main'
+    const userOrPublicKey = 'user'
+    const pwdOrPrivateKey = 'password'
+    const gitOrHelmAuth = 'Helm'
+    const gitAuthType = "http"
+    var helmUrlRegex
+
+    const privateHelmData: testData[] = [
+      { qase_id: 64,
+        repoName: "local-private-helm-repo-64",
+        path: 'helm-urlregex-repo',
+        test_explanation: 'repo',
+        helmUrlRegex_matching: '^http.*',
+      },
+      { qase_id: 65,
+        repoName: "local-private-helm-chart-65",
+        path: 'helm-urlregex-chart',
+        test_explanation: 'chart',
+        helmUrlRegex_matching: '^http.*app.*tgz$',
+      },
+    ]
+
+    // Actually just a preparation step
+    it("Prepare the private helm registry", { tags: '@preparation' }, () => {
+      cy.importYaml({ clusterName: 'local', yamlFilePath: 'assets/helm-server-with-auth-and-data.yaml' });
+      cy.nameSpaceMenuToggle('default');
```

Comment:
Mind please reminding me why toggling to default is needed here instead to local?

Reply:
ok. just saw not so important. Feel free to leave it as this if you wish.

Reply:
Lets keep it for now.